testium-cookie.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var _ = require('lodash');
+var debug = require('debug')('testium-cookie');
+
+function encode(string) {
+  return new Buffer(string).toString('base64');
+}
+
+function decode(value) {
+  return new Buffer(value, 'base64').toString('utf8');
+}
+
+function encodeMetaData(headers, statusCode) {
+  var jsonData = JSON.stringify({
+    headers: headers,
+    statusCode: statusCode
+  });
+  var encodedData = encode(jsonData);
+  return encodedData;
+}
+exports.encodeMetaData = encodeMetaData;
+
+function buildCookie(headers, statusCode) {
+  var jsonData = JSON.stringify({
+    headers: headers,
+    statusCode: statusCode
+  });
+  var encodedData = encode(jsonData);
```

Comment:
It looks like `encodedData` and `jsonData` variables aren't required since its purpose is fulfilled by `encodeMetaData`?